src/utils/prepare-detailed-product.ts

```diff
@@ -15,10 +15,23 @@ export const prepareDetailedProduct = (
       )
     : DEFAULT_PRICE;
 
+  const attributes = data.masterVariant?.attributes || [];
+
+  const getAttributeValue = (
+    attrName: string
+  ): Record<string, string> | string | number | boolean | undefined =>
+    attributes.find(attr => attr.name === attrName)?.value;
```

Comment:
```suggestion
    attributes.find(attr => attr.name === attrName)?.value || "";
```
уже так тогда, чтоб 3 раза не писать внизу